pipelines/install_pipeline.yml

```diff
@@ -25,10 +25,11 @@
     - role: disable_firewall
     - role: foreman_server_repositories
     - role: update_os_packages
-    - role: foreman
```

Comment:
If you no longer include these, you can drop the `foreman_installer_skip_installer: true` part on line 15.

Reply:
In https://github.com/theforeman/forklift/blob/master/pipelines/upgrade_pipeline.yml we do include the foreman-installer role so we get the `yum install` part of both boxes in parallel. I'd like to have both pipelines follow the same pattern if possible